metrics/firebase/test/firestore/rules/builds-test.js

```diff
@@ -206,24 +206,66 @@ describe("", async () => {
       );
     });
 
-    it("does not allow creating builds if the duration is null", async () => {
+    it("allows creating builds with a finished build status and a valid duration", async () => {
```

Comment:
I think it would be better to test that our rules allow creating a build document with not-null integer duration for each `finished` build status. What do you think?
```suggestion
    it("allows creating builds with a successful build status and non-null integer duration", async () => {
```